steps/promises/test.spec.js

```diff
@@ -1,39 +1,41 @@
-const {exercise1Fn, exercise2Fn, exercise3Fn} = require('./exo.js');
+const { exercise1Fn, exercise2Fn, exercise3Fn } = require('./exo.js');
 
 describe('Test exercise for promises', function () {
-
-  let instanceToTest;
+  let spyConsoleLog = undefined;
```

Comment:
C'est quoi l'apport de passer sur la syntaxe Jest ?

Reply:
Nothing in terms of performance or in the same area, it's just to be coherent with documentation and stackoverflow doc